src/app/public/modules/country-field/country-field.component.html

```diff
@@ -0,0 +1,59 @@
+<div
+  class="sky-form-group sky-input-group"
+>
+  <span
+    class="sky-country-field-container"
+    [ngClass]="{
+      'sky-country-field-disabled': disabled
+    }"
+  >
+    <form
+      novalidate
+      class="sky-country-field-country-search"
+      [formGroup]="countrySearchForm"
+    >
+      <sky-autocomplete
+        [data]="countries"
+        [propertiesToSearch]="['name', 'iso2']"
+        [searchResultTemplate]="countrySearchResultTemplate"
+        [searchTextMinimumCharacters]="2"
```

Comment:
Should we make `2` a variable so we could leverage that easily? Also, it could be easier to document.

Reply:
Done